### DISCUSSION TITLE
I am install but not work

### DISCUSSION THREAD

**Discussion by jagdish10max** [projectdiscovery/shuffledns#155] [Q&A] [answered]
I am install  go install -v github.com/projectdiscovery/shuffledns/cmd/shuffledns@latest

its did downloaded but ....

shuffledns -h 
just run >>shuffledns: command not found

I am also try..git clone and  $ GO111MODULE=on go get -v github.com/projectdiscovery/shuffledns/cmd/shuffledns

anyone advice mi

**Comment by 0xCGonzalo** [comment, marked as answer]
Here is a possible solution:

https://github.com/projectdiscovery/shuffledns/discussions/78

`go get` downloads packages under `GOPATH`, or if the environmental variable is not set I believe it defaults to `$HOME/go`, hence you should be looking there. (e.g. `$GOPATH/bin/shuffledns` or `~/go/bin/shuffledns`)

For the cloned repository:

```
cd shuffledns/cmd/shuffledns
go build -o shuffledns
chmod +x shuffledns
```
./shuffledns
